src/Framework/App.Runtime/src/Directory.Build.props

```diff
@@ -0,0 +1,7 @@
+<Project>
+  <PropertyGroup>
+    <OutDirName Condition="'$(MSBuildProjectName)' == 'Microsoft.AspNetCore.App.Runtime'">aspnetcore-sfx</OutDirName>
+    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
```

Comment:
Why in a Directory.Build.props? Why not just in the `Microsoft.AspNetCore.App.Runtime` directly? Does the OutDirName need to be set before other props files get imported?

Reply:
Yes, `OutDirName` must be set before the .NET SDK or the Arcade SDK is included.